packages/block-library/src/image/image-editing/use-transform-image.js

```diff
@@ -12,6 +18,11 @@ function useTransformState( { url, naturalWidth, naturalHeight } ) {
 	const [ aspect, setAspect ] = useState();
 	const [ defaultAspect, setDefaultAspect ] = useState();
 
+	const { imageEditingCrossOriginValue } = useSelect( ( select ) => {
+		const { getSettings } = select( 'core/block-editor' );
+		return pick( getSettings(), [ 'imageEditingCrossOriginValue' ] );
```

Comment:
I don't think we need `pick` here, checking for a `string` is probably sufficient. [The actual allowed values are only a few](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/crossOrigin).

```suggestion
		const value = select( 'core/block-editor' ).getSettings().imageEditingCrossOrigin;
		return typeof value === 'string' ? value : undefined;
```

Reply:
Yeah, I had that on my list, I noticed `pick` would probably not be needed for a single value, though I was focused on making it work for the very first iteration. This change looks good. Thanks for the nudge!